models.py

```diff
@@ -118,26 +118,18 @@ class SampledBallot(db.Model):
 
     # this ballot position should be 1-indexed
     ballot_position = db.Column(db.Integer, nullable=False)
+
+    ticket_number = db.Column(db.String(200), nullable=False)
     
     __table_args__ = (
-        db.PrimaryKeyConstraint('round_id', 'jurisdiction_id', 'batch_id', 'ballot_position'),
+        db.PrimaryKeyConstraint('round_id', 'jurisdiction_id', 'batch_id', 'ballot_position', 'ticket_number'),
     )
     
     times_sampled = db.Column(db.Integer, nullable=False)
```

Comment:
@umbernhard this is the field I'm worried about. Do we still need it in the DB now that we have a row per sample?

Reply:
oooohhhh. No, I think we can remove it. Right @eventualbuddha?

Reply:
Oh I see what's happening. We need to keep track of the total number of times a ballot was sampled so we can tell the Sampler where to pick up its sample. So removing this field breaks that.

Reply:
right, so that's where the inconsistency could sneak in. I think we're in agreement that, with this new model, the total number of times a ballot was sampled is the number of rows with a given `(jurisdiction_id, batch_id, position)`?

Reply:
Yeah, that's right.

Reply:
@umbernhard so it seems to me that we should remove this field and make sure that, when we need it, we select it from the database. I can take a look if you prefer?

Reply:
Yes please! Sorry, I thought we’d already talked about it but I must be misremembering.